tests/functional/swift-skip-metadata.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+RET=0
+set -e
+
+ACCOUNT=$(openio cluster list account -c Addr -f value)
+openio cluster lock account ${ACCOUNT}
+sleep 5
+
+# prepare object
+curl -XPUT http://localhost:5000/example/pass --data-binary @/etc/magic
+
+# read object
+curl -XPUT http://localhost:5000/example/pass --data-binary @/etc/magic
+
+# write object
+curl -XPUT http://localhost:5000/example/pass --data-binary @/etc/magic
```

Comment:
The previous 3 commands are the same, but with different comments. This seems not wanted.